js/ui/discoveryFeedButton.js

```diff
@@ -38,7 +38,10 @@ function _checkIfDiscoveryFeedEnabled() {
 function maybeCreateButton() {
     if (_checkIfDiscoveryFeedEnabled()) {
         let discoveryFeedButton = new DiscoveryFeedButton();
-        discoveryFeedButton.connect('clicked', Lang.bind(this, function() {
+        discoveryFeedButton.tile.connect('clicked', Lang.bind(this, function() {
```

Comment:
Not a big deal, but I think we should start arrow functions whenever possible instead of the -now no longer needed- Lang.bind() idiom